draft-ietf-quic-http.md

```diff
@@ -447,13 +497,12 @@ superseded by QUIC transport parameters in HTTP-over-QUIC. Below is a listing of
 how each HTTP/2 SETTINGS parameter is mapped: 
 
   SETTINGS_HEADER_TABLE_SIZE:
-  : Sent in SETTINGS frame.
+  : An integer with a maximum value of 2^32 - 1.
 
   SETTINGS_ENABLE_PUSH:
-  : Sent in SETTINGS frame (TBD, currently set using QUIC "SPSH" connection
-    option)
+  : A Boolean
```

Comment:
This is unfortunate.  Your change to booleans has made this have a new default.

Nit: missing period.